mldsa/src/poly.c

```diff
@@ -142,6 +142,19 @@ void mld_poly_shiftl(mld_poly *a)
   mld_assert_bound(a->coeffs, MLDSA_N, 0, MLDSA_Q);
 }
 
+static void mld_poly_ntt_c(mld_poly *a)
```

Comment:
To be able to implement the unit tests later the `static` need to be changed to `MLD_STATIC_TESTABLE`. Maybe we already do that now so it does not have to be changed later.